lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -258,74 +258,116 @@ def _get_converged_core_config(service_mconfig: object) -> bool:
 
 
 def _get_default_slice_service_type_config(service_mconfig: object) -> str:
-    """
-    Retrieves default_slice_service_type config value,If it does not exist
-    it defaults to 1.
+    """Retrieve default_slice_service_type config value. If it does not exist, it defaults to 1.
```

Comment:
let's not use numerical values on function documentation
```suggestion
    """Retrieve default_slice_service_type config value. If it does not exist, it defaults to DEFAULT_NGAP_S_NSSAI_SST value.
```

Reply:
Comment addressed.